helm/configurations/values-dev.yaml

```diff
@@ -31,3 +33,36 @@ components:
 
 pgbouncer:
   enabled: true
+
+# OpenSearch configuration for dev environment
+opensearch:
+  tls:
+    generate: false
+  singleNode: true
+  config:
+    opensearch.yml: |
+      cluster.name: opensearch-cluster
+      network.host: 0.0.0.0
+      plugins.security.disabled: true
+  extraEnvs:
+    - name: OPENSEARCH_INITIAL_ADMIN_PASSWORD
+      value: ReanaOS1=
```

Comment:
It would be better not to hard-code any password values in `values-dev.yaml`.

See also how we handle e.g. cache password in the 0.9.4:

```console
$ rg -C 3 REANA_CACHE_PASSWORD
• helm/reana/templates/reana-cache.yaml
49-      - name: cache
50-        image: docker.io/library/redis:5.0.5
51-        env:
52:          - name: REANA_CACHE_PASSWORD
53-            valueFrom:
54-              secretKeyRef:
55-                name: {{ include "reana.prefix" . }}-cache-secrets
56-                key: password
57-        args:
58-          - "--requirepass"
59:          - "$(REANA_CACHE_PASSWORD)"
60-        ports:
61-        - containerPort: 6379
62-      {{- if .Values.node_label_infrastructure }}

• helm/reana/templates/reana-server.yaml
151-              secretKeyRef:
152-                name: {{ include "reana.prefix" . }}-secrets
153-                key: REANA_SECRET_KEY
154:          - name: REANA_CACHE_PASSWORD
155-            valueFrom:
156-              secretKeyRef:
157-                name: {{ include "reana.prefix" . }}-cache-secrets
```